pydis_site/apps/content/resources/guides/python-guides/windows/putting-python-on-path.md

```diff
@@ -0,0 +1,254 @@
+will still need to locate the installer exe at step 5 if it's not already in Downloads.)
+
+### Verifying your changes
+
+To check that it worked, open a fresh terminal &mdash; Command Prompt, PowerShell, an IDE-integrated terminal, [Windows
+Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701), whichever you prefer, just _not_ the
+Python terminal with the `>>>` prompt &mdash; and run
+
+```text
+python -V
+```
+
+and then
+
+```text
+pip -V
+```
+
+and you should see the versions of Python and pip that you just added to Path:
+
+[![Checking Python and pip versions.](/static/images/content/python-on-windows/testing_path_worked_1.png)](/static/images/content/python-on-windows/testing_path_worked_1.png)
+
+Running `Get-Command python` in PowerShell, or `where python` in Command Prompt, you
+can see where that terminal is finding the `python.exe` it would run when a `python` command is given:
+
+[![Get-Command/where Python.](/static/images/content/python-on-windows/testing_path_worked_2.png)](/static/images/content/python-on-windows/testing_path_worked_2.png)
+
+(`where` in fact lists all the Pythons it finds, even the dummy [app execution
+alias](../microsoft-store#confusing-app-execution-alias-behaviour) one that opens the Microsoft Store.)
+
+You can also run Python with `python` (not `py` as that may start a different version, see more
+[here](../installing-and-using-python/#the-py-launcher)), and then after `>>>` in the Python REPL, run
+
+```py
+import sys; print(sys.executable); exit();
+```
+
+to see that the exact executable that is currently running Python is what you expect:
+
+[![Checking Python executable.](/static/images/content/python-on-windows/testing_path_worked_3.png)](/static/images/content/python-on-windows/testing_path_worked_3.png)
+
+Of course your username will probably not be "r", and your executable path may differ from
+`C:\Users\<user>\AppData\Local\Programs\Python\Python311` depending on where you chose to install Python and what
+version you have. The last folder will normally be `Python310` for Python 3.10, `Python39` for Python 3.9, etc. But
+those differences don't matter if `python` and `pip` and other things like `pyinstaller` (if you've [pip
+installed](https://pypi.org/project/pyinstaller/) it) now work for you!
+
+Hopefully things are indeed working, however, it is possible that, due to having multiple Python versions, or other
+mixups, the commands are still not behaving how you expect. If so, read on to learn how to manually edit the Path. (If
+you already did that, open a help channel on the [Python Discord server](https://discord.com/invite/python) and someone
+will hopefully be able to help.)
+
+## Advanced Method: Manually edit the Path
+
+### About the Path
+
+As mentioned above, the Path (often called "PATH", though it shows up as "Path") is what Windows uses to locate
```

Comment:
I'd used PATH (caps) previously throughout to clearly distinguish between it and a file/folder path, but I don't really mind.

Reply:
Yeah, I only used "Path" (and usually "the Path") since in the actual environment variables that's the case it shows up as, though a few times I use "PATH" since the python installer calls it that.

I wish the env var was uppercase PATH since then it is more distict and matches how it's usually talked about online.